vip_spec.xsd

```diff
@@ -531,7 +531,10 @@
   <xs:complexType name="PollingLocation">
     <xs:sequence>
       <!-- It is intentional that an address is required. -->
-      <xs:element name="AddressLine" type="xs:string" maxOccurs="unbounded" />
+      <xs:choice>
+        <xs:element name="AddressStructured" type="SimpleAddressType"/>
```

Comment:
While I *think* this will work, could you try and do XML validation on an old 5.1 file using this proposed XSD?

Reply:
The `sample_feed.xml` in this PR does validate against the XSD and there is 1 PollingLocation with a AddressStructured in that file.

Reply:
There are also PollingLocation records with unstructured addresses in sample_feed.xml, so I think we're good.